.github/workflows/dev-ci-cd.yml

```
@@ -0,0 +1,99 @@
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ env.AWS_REGION }}
+
+      - name: Deploy to EC2 instance via SSM Run Command
+        env:
+          IMAGE_URI: ${{ needs.build-and-push.outputs.image_uri }}
+          ECR_REGISTRY: ${{ needs.build-and-push.steps.login-ecr.outputs.registry }}
```

actionlint (1.7.7)

property "steps" is not defined in object type {outputs: {image_uri: string}; result: string}

(expression)